dashboard/views.py

```diff
@@ -227,7 +228,71 @@ def grade_distribution(request):
     return HttpResponse(df_grade.to_json(orient='records'))
 
 
+def assignment_view(request):
+    logger.info(assignment_view.__name__)
+
+    percent_selection = float(request.GET.get('percent','0.0'))
+    logger.info("selection from assignment view",percent_selection)
+    sql = "select assignment_id,local_graded_date as graded_date,score,name,local_date as due_date,points_possible,group_points,weight from (" \
+          "(select assignment_id,local_graded_date,score from"\
+          "(select id from USER where sis_name = %(current_user)s ) as u join"\
+          "(select user_id,assignment_id,local_graded_date,score from SUBMISSION) as sub on sub.user_id=u.id) as rock join"\
+          "(select assign_id,name,local_date,points_possible,group_points,weight from"\
+          "(select id as assign_id,assignment_group_id, local_date,name,points_possible from ASSIGNMENT where course_id = %(course_id)s) as a join"\
+          "(select id,group_points, weight from ASSIGNMENT_GROUPS) as ag on ag.id=a.assignment_group_id) as bottom on rock.assignment_id = bottom.assign_id)"
+    df = pd.read_sql(sql,conn,params={"current_user": current_user,'course_id': UDW_COURSE_ID},parse_dates={'due_date': '%Y-%m-%d','graded_date':'%m/%d/%Y'})
```

Comment:
We need to pass course id as a parameter in the next version. It is hard coded now for both file and assignment view. I will create an issue for this.